src/main/scala/li/cil/oc/integration/appeng/NetworkControl.scala

```diff
@@ -86,11 +93,12 @@ trait NetworkControl[AETile >: Null <: TileEntity with IGridProxyable with IActi
     })
   }
 
-  @Callback(doc = "function():table -- Get a list of the stored fluids in the network.")
+  // NOT Supported right now
+  /*@Callback(doc = "function():table -- Get a list of the stored fluids in the network.")
   def getFluidsInNetwork(context: Context, args: Arguments): Array[AnyRef] =
     result(tile.getProxy.getStorage.getFluidInventory.getStorageList.filter(stack =>
-      !Mods.ExtraCells.isAvailable || ECUtil.canSeeFluidInNetwork(stack)).
-      map(_.getFluidStack).toArray)
+      !Mods.ExtraCells.isModAvailable || ECUtil.canSeeFluidInNetwork(stack)).
```

Comment:
ECUtil is excluded and not available for compiling, and also Applied Energistics on it's own doesn't support Fluids